tests/integration_test.py

```diff
@@ -1624,3 +1624,6 @@ def test_649(self):
         ctnr = self.client.create_container('busybox', ['sleep', '2'])
         self.client.start(ctnr)
         self.client.stop(ctnr)
+
+    def test_715(self):
```

Comment:
Sure. My reasoning was that those tests won't be read unless there's an issue (regression) happening, in which case the person examining it will probably go to the original issue anyway. If you think there's value in renaming them that's fine too.